spec/spec_helper.rb

```diff
@@ -8,3 +8,15 @@ def getdata(name)
 def getform(name)
   FML::Form.new(getdata(name))
 end
+
+RSpec.configure do |config|
+# The settings below are suggested to provide a good initial experience
+# with RSpec, but feel free to customize to your heart's content.
+
+  # These two settings work together to allow you to limit a spec run
+  # to individual examples or groups you care about by tagging them with
+  # `:focus` metadata. When nothing is tagged with `:focus`, all examples
+  # get run.
+  config.filter_run :focus
```

Comment:
I usually don't commit it, but I like being able to add `focus: true` to specs to run just them. Can be removed.